pkg/operator/obj/webhooktriggerdeps.go

```diff
@@ -241,15 +242,26 @@ func (wtd *WebhookTriggerDeps) AnnotateTriggerToken(ctx context.Context, target
 	}
 
 	annotations := wtd.WebhookTrigger.Object.GetAnnotations()
+	labels := wtd.WebhookTrigger.Object.GetLabels()
 
 	claims.KubernetesNamespaceName = wtd.TenantDeps.Namespace.Name
 	claims.KubernetesNamespaceUID = string(wtd.TenantDeps.Namespace.Object.GetUID())
 	claims.KubernetesServiceAccountToken = sat
 
+	// FIXME This annotation is not being set right now ...
 	claims.RelayDomainID = annotations[model.RelayDomainIDAnnotation]
-	claims.RelayTenantID = annotations[model.RelayTenantIDAnnotation]
+
+	// FIXME This annotation is not being set right now ...
```

Comment:
Can we just fix this upstream in the tenant code or API code?

Reply:
So I avoided this for a few reasons. We're going to try to improve this handling on the API side. I don't like the options there either, as we're still working around the issue in a sense, but it is a better option regardless.